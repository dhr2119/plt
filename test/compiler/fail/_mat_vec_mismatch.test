int main() {
  matrix int[2,2] m;
  vector int[2] v;
  m = [1,2;3,4];
  v = | 1 | 2 |;
  m = m * v;
  return 0;
}