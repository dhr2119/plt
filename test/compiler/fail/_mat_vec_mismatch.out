Fatal error: exception Exceptions.AssignmentTypeMismatch("matrix", "vector")
